pkg/cli/proxy_get.go

```diff
@@ -0,0 +1,71 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"github.com/pkg/errors"
```

Comment:
You nee 3 groups of imports for the linters to pass - 1st is Go libraries,  2nd is 3rd party, 3rd is OSM local packages.